components/Button.tsx

```diff
@@ -0,0 +1,54 @@
+import Link from 'next/link';
+import PropTypes from 'prop-types';
+
+export const Button = ({
+  text,
+  defaultClass,
+  hoverClass,
+  href,
+  isExternal,
+}: {
+  text: string;
+  defaultClass: string;
+  hoverClass: string;
+  href: string;
+  isExternal: boolean;
+}) => {
+  const buttonClass =
+    'mx-auto px-2 py-3 border whitespace-nowrap transition duration-300 ease-in-out hover:';
```

Comment:
could you check if it's possible to show cursor pointer when hovering over a button?